index.js

```diff
@@ -67,6 +71,54 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+    if (header.style.color === 'black'){
+        header.style.color = 'red'
+    } else {
+        header.style.color = 'black'
+    }
+    
+})
+// didnt see toggleColor() when I originally wrote the function uptop. I'd basically take
+// that and send it into the listener method...please verify if the following would be correct.
+// addEventListner ('click',toggleColor)
 /***** Deliverable 2 *****/
 
+let likesElement = document.querySelector("#profile > p")
+let buttonElement = document.querySelector("#profile > button")
+buttonElement.addEventListener('click', function() {
+    likesInt = parseInt(likesElement.innerText.split(" ")[0],10)
+    likesInt += 1
+    likesElement.innerText = `${likesInt} Likes`
+})
+
 /***** Deliverable 3 *****/
+
+let formElement = document.querySelector("#new-animal-sighting-form")
+let sightingsList = document.querySelector("#animals")
+
+formElement.addEventListener("submit", function(event) {
+     event.preventDefault()
+
+     let speciesInput = event.target.species.value
+     let linkInput = event.target.link.value
+     let photoInput = event.target.photo.value
+     let descriptionInput = event.target.description.value
+     let animalId = sightingsList.children.length + 1
+
+     infoArray = [speciesInput, linkInput, photoInput, descriptionInput, animalId]
+     singleSighting(infoArray)
+})
+
+function singleSighting(object) {
+
+    let sightingInstance =  document.createElement('li')
+
+    sightingInstance.innerHTML = `<li data-id=${infoArray[4]}>
```

Comment:
You're getting a double `li` here because on line 115, you create an `li` for each sighing element, and then on 117 you declare that the `innerHTML` of that `li` will have another `li` !

If you wanted to use the innerHTML approach, you could select a `<ul>` and then set the innerHTML of **that** element like how you're doing on line 117.

Reply:
Or to avoid the innerHTML problem all together, you could do something like this:

```javascript
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);
function handleFormSubmission(e){
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species: species,
        videoLink: videoLink,
        photo: photo,
        description: description
    }
    renderAnimalSightingPost(newData);
}
sightingForm.addEventListener("submit", handleFormSubmission)
```